CHANGELOG.md

```diff
@@ -12,6 +13,17 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+## v0.4.2
+
+API version: v1alpha2
```

Comment:
didn't we add a bunch of stuff as well? like https://github.com/kubernetes-sigs/gateway-api/pull/1002/files

Reply:
Good question, starting with 0.4.0, we've branched off to allow for patches independent of what's being merged to master: https://github.com/kubernetes-sigs/gateway-api/commits/release-0.4